module-4/backend/week_one.md

```diff
@@ -5,18 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+* The idiosyncrasies and how they differ from Ruby.
+
 2. What are some tools to help debug JavaScript code?
+
+* debugger, pryjs, console.log
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+* mocha, chai
+
 4. What is the syntax for invoking a function?
+
+* ()
+
 5. What is CORS?
+
+* Cross-Origin Resource Sharing
+
 6. How do we enable CORS?
+
+* In the configuration of RACK CORS middleware to allow certain methods. From the requester, requires adding a headers object to the request
+
 7. What's `this` in JavaScript?
+
+* Refers to the object that the function is being acted upon
```

Comment:
you put it so simply here, but yet it gets complex/confusing quickly depending on where you are!